kore/src/Kore/ASTPrettyPrint.hs

```diff
@@ -303,9 +293,8 @@ instance
             ]
 
 instance
-    ( PrettyPrint child
-    , MetaOrObject level
-    ) => PrettyPrint (Application level child)
+    PrettyPrint child =>
```

Comment:
We should format these consistently (see the instance at line 279, I personally prefer that formatting, which seems more consistent with the other multi-line formatting cases in this file).